### PR TITLE
Added JSON_PRETTY function with tests & docs update

### PR DESCRIPTION
**Pull request by JulienCabanes**

### CODE REVIEW

File: src/Query/AST/Functions/Mysql/JsonPretty.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Platforms\MySqlPlatform;
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
+
+/**
+ * "JSON_PRETTY" "(" string ")"
```
Comment (Hikariii) [inline src/Query/AST/Functions/Mysql/JsonPretty.php]: Can you make sure the comment matches the parser function used.
In this way it is clear what the DQL parser does.
Putting `NewValue` in stead of `string` makes that much clearer.
I see now that this needs to be changed for `JSON_QUOTE` too (https://github.com/ScientaNL/DoctrineJsonFunctions/blob/763df144cc21eaaefe1ba3dbc771eab8c4e0c8d2/src/Query/AST/Functions/Mysql/JsonQuote.php#L13)

File: tests/Query/Functions/Mysql/JsonPrettyTest.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+use Doctrine\ORM\Query\Expr;
+
+class JsonPrettyTest extends MysqlTestCase
+{
+    public function testJsonPretty()
+    {
+        $this->assertDqlProducesSql(
+            "SELECT JSON_PRETTY('\"null\"') from Scienta\DoctrineJsonFunctions\Tests\Entities\Blank b",
+            "SELECT JSON_PRETTY('\"null\"') AS sclr_0 FROM Blank b0_"
+        );
```
Comment (Hikariii) [inline tests/Query/Functions/Mysql/JsonPrettyTest.php]: Can you also add a test parsing more complex DQL?
SQL queries like `SELECT JSON_PRETTY(NULL);` and `SELECT JSON_PRETTY(CAST(CONCAT('[', '1,2,45,3', ']') AS JSON));` must also work.
Just testing `"null"` is a bit too small.

**Comment by HarmenM** [comment]
@JulienCabanes Is there a possibility to update your PR in order to get it merged?

**Comment by HarmenM** [comment]
Closing the PR due to inactivity

**Pull request closed without merging**